src/mon/Monitor.cc

```diff
@@ -2323,6 +2330,9 @@ void Monitor::get_mon_status(Formatter *f, ostream& ss)
   for (set<int>::iterator p = quorum.begin(); p != quorum.end(); ++p) {
     f->dump_int("mon", *p);
   }
+  if (!quorum.empty()) {
+    f->dump_stream("quorum_age") << (mono_clock::now() - quorum_since);
```

Comment:
we'd better put the `quorum_age` in the top-level dictionary instead of in the `quorum` array.  

```
$ echo '{"name":"a","rank":0,"state":"leader","election_epoch":4,"quorum":[0,1,"1.46757s"],"features":{"required_con":"549755813888","required_mon":[],"quorum_con":"4611087854031142911","quorum_mon":["kraken","luminous","mimic","osdmap-prune","nautilus"]},"outside_quorum":[],"extra_probe_peers":["127.0.0.1:7128/0","127.0.0.1:7129/0"],"sync_provider":[],"monmap":{"epoch":1,"fsid":"02ed5ad9-a397-4dc5-9c1a-0e5d9b2bea37","modified":"2018-09-04 10:03:38.314078","created":"2018-09-04 10:03:38.314078","features":{"persistent":[],"optional":[]},"mons":[{"rank":0,"name":"a","addr":"127.0.0.1:7127/0","public_addr":"127.0.0.1:7127/0"},{"rank":1,"name":"b","addr":"127.0.0.1:7128/0","public_addr":"127.0.0.1:7128/0"},{"rank":2,"name":"c","addr":"0.0.0.0:0/2","public_addr":"0.0.0.0:0/2"}]},"feature_map":{"mon":[{"features":"0x3ffddff8ffa4ffff","release":"luminous","num":1}],"client":[{"features":"0x3ffddff8ffa4ffff","release":"luminous","num":1}]}}' | jq '.quorum'
[
  0,
  1,
  "1.46757s"
]
```

see http://pulpito.ceph.com/kchai-2018-09-04_09:15:50-rados-wip-kefu-testing-2018-09-04-1125-distro-basic-smithi/2978749/

Reply:
fixed!